rackspace/networking/v2/networks/delegate_test.go

```diff
@@ -203,8 +203,17 @@ func TestCreateWithOptionalFields(t *testing.T) {
 	}
 }
 		`)
-
 		w.WriteHeader(http.StatusCreated)
+		fmt.Fprintf(w, `
```

Comment:
@jrperritt so, this is an instance of what I was talking about. The test failed here because the patch returns an error if the response is empty (`unexpected error "EOF"`) but this change fixes the test.